packages/commonwealth/client/scripts/views/pages/create_community/starter_community_form.tsx

```diff
@@ -37,6 +39,25 @@ export const StarterCommunityForm = () => {
 
   const navigate = useCommonNavigate();
 
+  useEffect(() => {
+    const selectChainForBase = async () => {
```

Comment:
Does this really need to be async?

Reply:
nope, good catch. It was async in a previous commit. Fixed.